### PR TITLE
Let Ansible use Python3 on remote

### PR DESCRIPTION
**Pull request by tomwassenberg**
Ansible has stable Python 3.5+ support since Ansible 2.5, so we don't need Python 2.7 any more.

**Should not be merged before #175 is and we're creating a new prod environment.**

### CODE REVIEW

File: ansible/hosts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [production]
-svsticky.nl ansible_python_interpreter=/usr/bin/python2.7
+svsticky.nl ansible_python_interpreter=/usr/bin/python3
```
Comment (duijf) [inline ansible/hosts]: Do we need python3 or python3.5?

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ server to become Sticky's production server.
 
 The code in this repository depends on the following software:
 
- - [ansible] >= 2.4
+ - [ansible] >= 2.5
```
Comment (duijf) [inline README.md]: Do we want a `requirements.txt` or something based on pipenv?

File: ansible/hosts (hunk 1 above)
Comment (tomwassenberg) [reply ansible/hosts]: Err, Ansible needs 3.5+, and Ubuntu 18.04 ships with 3.6, so I'll change this to `python3.6`.

File: README.md (hunk 2 above)
Comment (tomwassenberg) [reply README.md]: We _could_, but in my opinion that's still a bit overkill at the moment.

**Pull request merged**